services/uploadToLocal.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
+const multer = require("multer");
+const { TEMP_DIR_VIDEOS } = require("../constants/paths");
+const ensureDir = require("../util/ensureDir");
+
+const uploadToLocal = multer({
+  storage: multer.diskStorage({
+    destination: async (req, file, done) => {
+      let destinationPath = "";
+
+      switch (file.fieldname) {
+        case "mainVideoFile":
+          destinationPath = TEMP_DIR_VIDEOS.MAIN;
+          break;
+        case "subOneVideoFile":
+          destinationPath = TEMP_DIR_VIDEOS.SUB_ONE;
+          break;
+        case "subTwoVideoFile":
+          destinationPath = TEMP_DIR_VIDEOS.SUB_TWO;
+          break;
+        default:
+          throw new Error(`Unknown fileName: ${file.fieldname}`);
+      }
+
+      await ensureDir(destinationPath);
+      done(null, destinationPath);
+    },
+    filename(req, file, done) {
+      let fileName = "";
+
+      switch (file.fieldname) {
+        case "mainVideoFile":
+          fileName = "main-video";
+          break;
+        case "subOneVideoFile":
+          fileName = "sub-one-video";
+          break;
+        case "subTwoVideoFile":
+          fileName = "sub-two-video";
+          break;
+        default:
+          throw new Error(`Unknown fileName: ${file.fieldname}`);
+      }
+
+      const ext = path.extname(file.originalname);
+      done(null, fileName + ext);
```

Comment:
```suggestion
      const ext = path.extname(file.originalname);
      
      done(null, fileName + ext);
```
할당문과 실행문사이에는 개행이 있으면 더 좋을 것 같아요~!